cli/generators/auth/index.js

```diff
@@ -0,0 +1,161 @@
+      this.destinationPath("app/components/auth/LoginForm.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('loginPage.js'),
+      this.destinationPath("app/components/auth/LoginPage.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('signUpForm.js'),
+      this.destinationPath("app/components/auth/SignUpForm.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('signUpPage.js'),
+      this.destinationPath("app/components/auth/SignUpPage.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('authReducer.js'),
+      this.destinationPath("app/reducers/authReducer.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('loginService.js'),
+      this.destinationPath("app/services/loginService.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('signUpService.js'),
+      this.destinationPath("app/services/signUpService.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('styles.css'),
+      this.destinationPath("app/styles/styles.css")
+    );
+
+    //The following files replace the ones already in the project
+    this.fs.copy(
+      this.templatePath('addApiRoutes.js'),
+      this.destinationPath("api/addApiRoutes.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('addApiRoutes.js'),
+      this.destinationPath("api/addApiRoutes.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('server.js'),
+      this.destinationPath("api/server.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('userSchema.js'),
+      this.destinationPath("api/user/user.schema.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('apiUserService.js'),
+      this.destinationPath("api/user/user.service.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('actionTypes.js'),
+      this.destinationPath("app/actions/actionTypes.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('app.js'),
+      this.destinationPath("app/components/App.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('header.js'),
+      this.destinationPath("app/components/common/Header.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('userForm.js'),
+      this.destinationPath("app/components/users/UserForm.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('appReducersIndex.js'),
+      this.destinationPath("app/reducers/index.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('initialState.js'),
+      this.destinationPath("app/reducers/initialState.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('routes.js'),
+      this.destinationPath("app/routes.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('apiEndpoints.js'),
+      this.destinationPath("app/services/apiEndpoints.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('appUserService.js'),
+      this.destinationPath("app/services/userService.js")
+    );
+
+    this.fs.copy(
+      this.templatePath('package.json'),
+      this.destinationPath("package.json")
+    );
+
+  }
```

Comment:
all these files have changes compared to the originals?

Reply:
All have changes, but not all are new files...
Taking the assumption that it will run immediately we download the seed, I just copied all the files to their target folder as a quick solution. We can of course improve this.

Reply:
ok, as we chat before, in the mean time we can implement the modifications for the files doing update

Reply:
Sure